airbyte-integrations/bases/base-normalization/integration_tests/dbt_integration_test.py

```diff
@@ -115,6 +118,7 @@ def setup_mysql_db(self):
         ]
         print("Executing: ", " ".join(commands))
         subprocess.call(commands)
+        time.sleep(120)
```

Comment:
This should not be here, it makes the test wait for two minutes before doing anything

Reply:
Sorry Chris! I add this because in my Ubuntu Setup the mysql and postgres took longer to start. I created this: https://github.com/airbytehq/airbyte/issues/6091